R/zchunk_L210.water.demand.primary.R

```diff
@@ -34,41 +35,55 @@ module_water_L210.water.demand.primary_DISABLED <- function(command, ...) {
     A22.globaltech_coef <- get_data(all_data, "energy/A22.globaltech_coef")
     L110.water_demand_primary_R_S_W_m3_GJ <- get_data(all_data, "L110.water_demand_primary_R_S_W_m3_GJ")
 
-    # ===================================================
-    # TRANSLATED PROCESSING CODE GOES HERE...
-    #
-    # If you find a mistake/thing to update in the old code and
-    # fixing it will change the output data, causing the tests to fail,
-    # (i) open an issue on GitHub, (ii) consult with colleagues, and
-    # then (iii) code a fix:
-    #
-    # if(OLD_DATA_SYSTEM_BEHAVIOR) {
-    #   ... code that replicates old, incorrect behavior
-    # } else {
-    #   ... new code with a fix
-    # }
-    #
-    #
-    # NOTE: there are `merge` calls in this code. Be careful!
-    # For more information, see https://github.com/JGCRI/gcamdata/wiki/Name-That-Function
-    # NOTE: there are 'match' calls in this code. You probably want to use left_join_error_no_match
-    # For more information, see https://github.com/JGCRI/gcamdata/wiki/Name-That-Function
-    # ===================================================
+
+    results <- tibble()
+
+    if(1){
+      A21.globaltech_coef %>% select(supplysector,subsector,technology) -> A21.tmp1
+       A22.globaltech_coef %>% filter( grepl('nuclear', supplysector)) %>% select(supplysector,subsector,technology) -> A22.tmp
+       A21.A22.tmp <- bind_rows(A21.tmp1,A22.tmp)
+       left_join(L110.water_demand_primary_R_S_W_m3_GJ, A21.A22.tmp, by="supplysector") -> L210.TechCoef.v1
+
```

Comment:
Lines above are pretty confusing to read. At the very least, be consistent about direction of assignment arrow and put different pipeline steps (`>%>`) on separate lines.